gen3/neural-networks/advanced-examples/image-matching/xfeat/utils/arguments.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+
+def initialize_argparser():
+    """Initialize the argument parser for the script."""
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "-d",
+        "--device",
+        help="Optional name, DeviceID or IP of the camera to connect to.",
+        required=False,
+        default=None,
+        type=str,
+    )
+
+    parser.add_argument(
+        "-fps",
+        "--fps_limit",
+        help="FPS limit for the model runtime.",
+        required=False,
+        default=30,
+        type=int,
+    )
+
+    parser.add_argument(
+        "-m",
+        "--model",
+        help="The XFeat model. Get it from the Luxonis HubAI.",
```

Comment:
maybe mention one needs to provide the HubAI model reference (ex-slug)

Reply:
Fixed in [2cb2f3e](https://github.com/luxonis/depthai-experiments/pull/562/commits/2cb2f3e6e90caf1f605cb618cdbe14ac89197f66).